### PR TITLE
Add a new company which comes from China

### PR DESCRIPTION
**Pull request by LinuxSuRen**
The name is Merico. The Chinese name is `思码逸`. See also https://www.merico.cn/our-story

### CODE REVIEW

**Comment by frank-zsy** [comment]
Thanks for the labeled data, I will add a script first to generate markdown format content from labeled data and add to GitHub actions to test PRs and automatically push to doc folder. Please hold this PR for a while.

**Comment by frank-zsy** [comment]
@LinuxSuRen sorry for close and reopen the PR several times for test. But I think I finally figured out the function to generate readable content from a label file in PR.

The test for actions is really painful and it can not be done automatically on PR open for 2 reasons:
-  Any forked repository PR triggered actions can not access secrets in actions, so it can not use my own token to create the comment.
- Any forked repository PR triggered actions will hold a default token with maximum access to PR read and can not create comment either due to the [permission policy](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token).

So we can use `\label-data-test` command to trigger the test process and the result will be commented back to the PR with a full content of yml files that changed in current PR. Is that looks good to you?

File: labeled_data/companies/merico/index.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Merico
+type: Company
+data:
+  github_org:
+    - 48216022 # merico-io
```
[the remark this answers is not in the document]
Comment (LinuxSuRen) [reply labeled_data/companies/merico/index.yml]: I thought they just renamed the name. Actually, they created a new one.

**Comment by LinuxSuRen** [comment]
hi @frank-zsy , it is ok to me.

There are two potential good ways to test GitHub Actions.

* using this command [act](https://github.com/nektos/act)
* Create a new repo to do that kind of test, this is pretty complex

**Comment by frank-zsy** [comment]
> hi @frank-zsy , it is ok to me.
> 
> There are two potential good ways to test GitHub Actions.
> 
> * using this command [act](https://github.com/nektos/act)
> * Create a new repo to do that kind of test, this is pretty complex

Thanks, it looks like `act` can be used to to test actions locally. And actually I test the actions on my own repo first and it looks good since I have full access to my own repo and PR from the same repository is not quite same from a different forked repository.

**Comment by LinuxSuRen** [comment]
No worries. It often happens.

**Comment by frank-zsy** [comment]
Thanks for the data and advise again.

/approve

**Pull request merged**